runtime/compilers/rillv1/parser.go

```diff
@@ -172,6 +173,11 @@ func Parse(ctx context.Context, repo drivers.RepoStore, instanceID string, duckD
 		return nil, err
 	}
 
+	// parse .env file separately
+	if err := p.parseDotEnv(ctx); err != nil {
+		return nil, err
+	}
```

Comment:
This is not sufficient – see how `rill.yaml` is handled:

1. Need to clear it in `Reparse` if path was changed 
2. Need to add bool for tracking modification to it in `Diff`
3. Should be parsed in `parsePaths` so it's parsed both in `Parse` and `Reparse`
4. Unlike `rill.yaml` (which returns an error), the returned error should be added to parse errors instead of being returned (like all other errors than `rill.yaml` errors)